integration-tests/authorizenet_test.go

```diff
@@ -173,4 +174,47 @@ func TestAuthNetAuthVoid(t *testing.T) {
 	}
 }
 
-// TestAuthNetAuthCaptureRefund cannot be written since there is no way to hack settlement and funds cannot refund until they settle
+// TestAuthNetAuthCaptureRefund
+func TestAuthNetAuthCaptureRefund(t *testing.T) {
```

Comment:
Why perform an auth and capture before the refund if we are just refunding directly against a credit card rather than refunding a specific transaction?

Reply:
still need a valid transactionReference if we want to test authnet accepting that field

Reply:
Are we planning to test that later? It doesn't look like we pass it in the refundRequest currently.

Reply:
yeah we will once automatic settlement is possible